lib/octopus.rb

```diff
@@ -106,10 +106,18 @@ def self.rails51?
     ActiveRecord::VERSION::MAJOR == 5 && ActiveRecord::VERSION::MINOR == 1
   end
 
+  def self.rails52?
+    ActiveRecord::VERSION::MAJOR == 5 && ActiveRecord::VERSION::MINOR == 2
+  end
+
   def self.atleast_rails51?
     ActiveRecord::VERSION::MAJOR > 5 || (ActiveRecord::VERSION::MAJOR == 5 && ActiveRecord::VERSION::MINOR >= 1)
   end
 
+  def self.atleast_rails52?
+    ActiveRecord::VERSION::MAJOR > 5 || (ActiveRecord::VERSION::MAJOR == 5 && ActiveRecord::VERSION::MINOR >= 1)
```

Comment:
Guys, since the method name is `atleast_rails52`, shouldn't it be:
```rb
ActiveRecord::VERSION::MAJOR >= 5 && ActiveRecord::VERSION::MINOR >= 2
# or Fix ActiveRecord::VERSION::MINOR >= 1 to ActiveRecord::VERSION::MINOR >= 2
# because it's exactly the same as the `.atleast_rails51?` method is.
```
The same question for the method `. atleast_rails51?`...
🤔

Reply:
@thiagopradi ☝️

Reply:
@bvicenzo - you are right. I'll double check and fix this method.